spiceypy/tests/test_wrapper.py

```diff
@@ -6699,30 +6768,36 @@ def test_srfxpt():
     spice.kclear()
     # load kernels
     spice.furnsh(CoreKernels.testMetaKernel)
-    spice.furnsh(MarsKernels.mgsSclk)
-    spice.furnsh(MarsKernels.mgsPck)
-    spice.furnsh(MarsKernels.mgsIk)
-    spice.furnsh(MarsKernels.mgsSpk)
-    spice.furnsh(MarsKernels.mgsCk)
+    spice.furnsh(CassiniKernels.cassSclk)
+    spice.furnsh(CassiniKernels.cassFk)
+    spice.furnsh(CassiniKernels.cassPck)
+    spice.furnsh(CassiniKernels.cassIk)
+    spice.furnsh(CassiniKernels.cassSclk)
+    spice.furnsh(CassiniKernels.satSpk)
+    spice.furnsh(CassiniKernels.cassTourSpk)
+    spice.furnsh(CassiniKernels.cassCk)
     # start test
-    et = spice.str2et("2006 OCT 15 06:00:00 UTC")
-    camid = spice.bodn2c("MGS_MOC_NA")
+    et = spice.str2et("2013 FEB 25 11:50:00 UTC")
+    camid = spice.bodn2c("CASSINI_ISS_NAC")
     shape, frame, bsight, n, bounds = spice.getfov(camid, 4)
-    # run srfxpt on bore sight vector
-    spoint, dist, trgepc, obspos = spice.srfxpt("Ellipsoid", 'Mars', et, "LT+S", "MGS", frame, bsight)
-    npt.assert_almost_equal(dist, 386.91994715)
-    npt.assert_almost_equal(trgepc, 214164065.18107578)
-    expected_spoint = [-1798.27290315, -629.0785937, -2794.96014468]
-    expected_obspos = [-2088.42506636, -718.58930162, -3034.7654842 ]
+    # run srfxpt on boresight vector
+    spoint, dist, trgepc, obspos = spice.srfxpt("Ellipsoid", 'Enceladus', et, "LT+S", "CASSINI", frame, bsight)
+    npt.assert_almost_equal(dist, 683459.6415073496)
+    npt.assert_almost_equal(trgepc, 415065064.9055491)
+    expected_spoint = [-143.56046006834264971985,  202.9004595420923067195,
+                       -27.99454299292458969717]
+    expected_obspos = [329627.25001832831185311079,  557847.97086489037610590458,
+                       -217744.02422016291529871523]
     npt.assert_array_almost_equal(spoint, expected_spoint)
     npt.assert_array_almost_equal(obspos, expected_obspos)
     # Iterable ET argument:  et-10, et, et+10
-    ets = [et - 10.0, et, et + 10.0]
-    sdtoArr = spice.srfxpt("Ellipsoid", 'Mars', ets, "LT+S", "MGS", frame, bsight)
+    ets = et - 10. + (np.arange(3) * 10.)
```

Comment:
use `ets = [et - 10.0, et, et + 10.0]` , and the len assert below can be removed

Reply:
The assert is just my AR; it can be removed anyway.  But what does it hurt?

Reply:
nothing, it is just not very helpful either

Reply:
fair enough, it's going away.